### PR TITLE
Correctly migrate to database schema v10

### PR DESCRIPTION
**Pull request by hughsie**
Artificially bump the schema version to v11 to ensure the migration is done.

Fixes https://github.com/fwupd/fwupd/issues/6812

Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [X] Code fix
- [ ] Feature
- [ ] Documentation

### CODE REVIEW

**Comment by hughsie** [comment]
Note: you only hit this when upgrading to 1.9.13 from earlier than 1.9.5 which is why we didn't hit it during testing.

**Comment by superm1** [comment]
Can we unit test big upgrade jump combos like this?

**Comment by hughsie** [comment]
> Can we unit test big upgrade jump combos like this?

Ya, we probably should; I'll have a go Monday. Do you think we should check in the .db files, or make a `pending.db` by using a schema of a specific version?

**Comment by superm1** [comment]
> > Can we unit test big upgrade jump combos like this?
> 
> Ya, we probably should; I'll have a go Monday. Do you think we should check in the .db files, or make a `pending.db` by using a schema of a specific version?

If it's easy enough I think make the db files on demand. Then do a jump up from each schema to latest in the tests and make sure it works.

**Comment by hughsie** [comment]
> Then do a jump up from each schema to latest in the tests and make sure it works

Ahh, we already had tests from v1 to the new latest schema, but we needed v2 to {latest} to actually test the migration. Coverage should be happier with us too. I've added the new test to match the old test.

File: src/fu-self-test.c [hunk 1]
```diff
@@ -3841,6 +3841,43 @@ fu_history_migrate_func(gconstpointer user_data)
...
+	g_autoptr(FuHistory) history = NULL;
+	g_autofree gchar *filename = NULL;
+
+#ifndef HAVE_SQLITE
+	g_test_skip("no sqlite support");
+	return;
+#endif
+
+	/* load old version */
+	filename = g_test_build_filename(G_TEST_DIST, "tests", "history_v2.db", NULL);
```
[the remark this answers is not in the document]
Comment (hughsie) [reply src/fu-self-test.c:3861]: I don't think we do _installed_ tests for src/* -- I'm not sure such a thing makes much sense either.

**Comment by hughsie** [comment]
> After backport I think this justifies a new release.

Ack.

**Pull request merged**